### ISSUE TITLE
Can we add web version ?

### ISSUE THREAD

**Issue by spectrum705** [daspartho/SpotiByeAds#67]
can we add a feature to use this in spotify web ? 
it will be very convenient. If possible i'll contribute too with whatever I can.
thank you,

**Comment by airD173** [comment]
I don't think that can be done. We could use Repl.it to monitor the songs and such on the web, but we something on the user's device to actually kill the Spotify process.

**Comment by airD173** [comment]
I deleted my old comment since I misunderstood the question. Can ads be stopped on Spotify web by closing the tab?

**Comment by spectrum705** [comment]
as for me, The ad disappears but so does the play button but it might work if we use the play_next_track() function on it. We'll have to check.
